Documentation/operations/upgrade.rst

```diff
@@ -309,6 +309,16 @@ Annotations:
 
 .. _1.13_upgrade_notes:
 
+1.13.1 Upgrade Notes
+--------------------
```

Comment:
What happens when we release 1.13.2? Ideally it would be clear whether v1.13.2 is also affected or not.

Reply:
Ah, I see that there is a PR out https://github.com/cilium/cilium/pull/24773 which is presumably expected to block the release. Perhaps we can make it clear after that is merged that the following release is not impacted by the issue.

Reply:
I do intend to update the upgrade guide on each stable branch once the PR is merged. But it's (unsurprisingly?) easier to get reviews on docs PR than on fix PRs...